### PR TITLE
added more guest os

### PR DESCRIPTION
**Pull request by karuturi**
centos 6.6, centos 6.7, rhel 6.6, rhel 6.7, windows server 2016, rhel
7.2, centos 7.2

This closes #1244

### CODE REVIEW

**Comment by rhtyd** [comment]
@karuturi this looks useful, can you get a JIRA id for this and amend the git commit summary with the ID. Can you also consider changing the PR's base branch to 4.9?

**Comment by rhtyd** [comment]
@karuturi ping

**Comment by rhtyd** [comment]
@karuturi ping

**Comment by rhtyd** [comment]
@karuturi @koushik-das ping

**Comment by sateesh-chodapuneedi** [comment]
@karuturi Can you please resolve the conflicts and rebase this branch?

File: setup/db/db/schema-4910to41000.sql [hunk 1]
```diff
@@ -45,4 +45,48 @@ CREATE TABLE `cloud`.`vlan_details` (
...
+INSERT IGNORE INTO `cloud`.`guest_os_hypervisor` (uuid,hypervisor_type, hypervisor_version, guest_os_name, guest_os_id, created, is_user_defined) VALUES (UUID(),'VMware', '6.0', 'rhel6_64Guest', 266, now(), 0);
+INSERT IGNORE INTO `cloud`.`guest_os_hypervisor` (uuid,hypervisor_type, hypervisor_version, guest_os_name, guest_os_id, created, is_user_defined) VALUES (UUID(),'VMware', '6.0', 'rhel6_64Guest', 268, now(), 0);
+
+INSERT IGNORE INTO `cloud`.`guest_os` (id, uuid, category_id, display_name, created) VALUES (269, UUID(), 6, 'Windows Server 2016 (64-bit)', now());
+INSERT IGNORE INTO `cloud`.`guest_os_hypervisor` (uuid, hypervisor_type, hypervisor_version, guest_os_name, guest_os_id, created) VALUES (UUID(), 'Xenserver', '7.0.0', 'Windows Server 2016 (64-bit)', 269, now());
+INSERT IGNORE INTO `cloud`.`guest_os_hypervisor` (uuid, hypervisor_type, hypervisor_version, guest_os_name, guest_os_id, created) VALUES (UUID(), 'VMware', '6.0', 'windows9srv-64', 269, now());
+
+INSERT IGNORE INTO `cloud`.`guest_os` (id, uuid, category_id, display_name, created) VALUES (270, UUID(), 4, 'Red Hat Enterprise Linux 7.2', now());
```
Comment (sateesh-chodapuneedi) [inline setup/db/db/schema-4910to41000.sql]: Though the official RHEL 7 distribution is 64-bit only, it'd be better to explicitly specify `"(64-bit)`" in the guest os display name. This is same as in vSphere client and other places where RHEL 7 shows up as `"Red Hat Enterprise Linux 7 (64-bit)"`

**Comment by sateesh-chodapuneedi** [comment]
@karuturi Can you please resolve the conflicts and rebase this branch?

**Comment by rhtyd** [comment]
@karuturi can you fix the conflicts and address outstanding issues; I can consider and help review/merge this for 4.9.2.0 target if you can fix them in next 2-3 days. Thanks.

**Comment by sureshanaparti** [comment]
@rhtyd Added more guest OS mappings and rebased against latest master.

**Comment by sateesh-chodapuneedi** [comment]
@sureshanaparti Seems build timed out. Can you please try force push?
```
[INFO] --- findbugs-maven-plugin:3.0.3:findbugs (findbugs) @ cloud-plugin-user-authenticator-md5 ---
[INFO] Fork Value is true
Build timed out (after 77 minutes). Marking the build as aborted.
Build was aborted
```

```
[INFO] Apache CloudStack Plugin - Power Management Driver ipmitool SUCCESS [ 34.609 s]
[INFO] Apache CloudStack Plugin - Storage Allocator Random  SUCCESS [ 26.572 s]
[INFO] Apache CloudStack Plugin - User Authenticator LDAP . SUCCESS [ 48.989 s]
[INFO] Apache CloudStack Plugin - User Authenticator MD5 .. FAILURE [ 10.151 s]
[INFO] Apache CloudStack Plugin - User Authenticator PBKDF2-SHA-256 SKIPPED
```

**Comment by rhtyd** [comment]
Ping, @karuturi can you squash your changes and rebase? Let's add the support in 4.10.

**Comment by luhaijiao** [comment]
can we have it too in 4.10 @karuturi

**Pull request merged**